frankenphp.go

```diff
@@ -345,6 +348,12 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	}
 
 	if rc != nil {
+		go func() {
```

Comment:
Couldn't we check if this channel has been closed only when needed (in `go_ub_write` and in `go_sapi_flush` instead? Creating as few goroutines as possible improves memory usage and scalability.

This will also remove the need for `fc.finished`.